README.md

```diff
@@ -51,6 +51,12 @@ emitter.on('foo', onFoo)   // listen
 emitter.off('foo', onFoo)  // unlisten
 ```
 
+### Typescript
+```ts
+import * as mitt from 'mitt';
+let emitter: mitt.Emitter = new mitt();
```

Comment:
@seveves yea, I think it would be better too. The idea is that `mitt` is emitter without context, so not make sense to call it with `new`.